lib/themes/reps/template.php

```diff
@@ -445,7 +445,7 @@ function reps_process_entity(&$variables) {
  * Implements template_preprocess_views_view_fields().
  */
 function reps_preprocess_views_view_unformatted(&$vars) {
-  if ($vars['view']->name == 'reps_bean_blocks_global') {
+  if ($vars['view']->current_display == 'block_reps_bean') {
```

Comment:
`if ($vars['view']->name == 'reps_bean_blocks_global' && $vars['view']->current_display == 'block_reps_bean') `
If you have different displays in your view and if you have multiple views with the same display name, you're covered. Sorry if it wasn't clear the first time.